docs/deployments.rst

```diff
@@ -40,23 +34,33 @@ such as changes that require re-indexing. See
 We deploy to production from ``master``.
 
 
-Dev
-===
+Deploying
+=========
```

Comment:
We can also mention here that we are currently using K8s and maybe something about conventions similar to what we documented in the processes document.